.github/workflows/cypress.yml

```
@@ -0,0 +1,108 @@
+    
+     - name: Run Cypress Tests with Dev Server
+       uses: cypress-io/github-action@v6
+       with:
+         working-directory: frontend
+         start: npm run serve
+         wait-on: 'http://localhost:4321'
+         wait-on-timeout: 120
+         config-file: cypress.config.ts
+         install: false
+       env:
+         CYPRESS_BASE_URL: http://localhost:4321
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 9

(indentation)